### ISSUE TITLE
re-Training from kitti and training on custom data leads to consistent black screen 

### ISSUE THREAD

**Issue by hamzajaved05** [nianticlabs/monodepth2#158]
Hey,
I am trying to retrain the Kitti model for my custom dataset (monocular RGB-only images in a sequence [0, 1]) and It converges to a black screen. Did you encounter an issue like this or was this ever experienced during training? I tried training from scratch (pre-trained ResNet weights) as well as retraining the Network trained on Kitti. Also, the loss logger shows a consistent loss of ~0.11  ~ .12 all the time. I figure there must be an issue with the losses that I need to modify for my data. I have a normalized K matrix.
The data I have is 8194 sequenced images from 4-5 settings. 
All other params are the basic ones.

**Comment by daniyar-niantic** [comment]
What is the resolution of images that you use for training? It is best if your aspect ratio matches original image aspect ratio (subject to divisibility by 32 requirement). Could you share some images of your sequences, it may help us understand what is going on.

**Comment by hamzajaved05** [comment]
The matter was resolved after fixing the camera matrix. Iirc the camera matrix had to be rescaled.
